ee/ucp/admin/configure/deploy-route-reflectors.md

```diff
@@ -1,137 +1,133 @@
-                  name: calico-config
-                  key: etcd_key # Location of the client certificate for etcd.
-            - name: ETCD_CERT_FILE
-              valueFrom:
-                configMapKeyRef:
-                  name: calico-config
-                  key: etcd_cert
-            - name: IP
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.podIP
-          volumeMounts:
-            - mountPath: /calico-secrets
-              name: etcd-certs
-          securityContext:
-            privileged: true
-      nodeSelector:
-        com.docker.ucp.kubernetes.calico/route-reflector: "true"
-      volumes:
-      # Mount in the etcd TLS secrets.
-        - name: etcd-certs
-          secret:
-            secretName: calico-etcd-secrets
-  ```
-
-Then, deploy the DaemonSet using:
-
-```
-kubectl create -f calico-rr.yaml
-```
+    updateStrategy:
+        type: RollingUpdate
+    selector:
+        matchLabels:
+        k8s-app: calico-rr
+    template:
+        metadata:
+        labels:
+            k8s-app: calico-rr
+        annotations:
+            scheduler.alpha.kubernetes.io/critical-pod: ''
+     spec:
+        tolerations:
+            - key: com.docker.ucp.kubernetes.calico/route-reflector
+            value: "true"
+            effect: NoSchedule
+         hostNetwork: true
+        containers:
+            - name: calico-rr
+            image: calico/routereflector:v0.6.1
+              env:
+                - name: ETCD_ENDPOINTS
+                  valueFrom:
+                    configMapKeyRef:
+                      name: calico-config
+                      key: etcd_endpoints
+                - name: ETCD_CA_CERT_FILE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: calico-config
+                      key: etcd_ca
+                # Location of the client key for etcd.
+                - name: ETCD_KEY_FILE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: calico-config
+                      key: etcd_key # Location of the client certificate for etcd.
+                - name: ETCD_CERT_FILE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: calico-config
+                      key: etcd_cert
+                - name: IP
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: status.podIP
+              volumeMounts:
+                - mountPath: /calico-secrets
+                  name: etcd-certs
+              securityContext:
+                privileged: true
+          nodeSelector:
+            com.docker.ucp.kubernetes.calico/route-reflector: "true"
+          volumes:
+          # Mount in the etcd TLS secrets.
+            - name: etcd-certs
+              secret:
+                secretName: calico-etcd-secrets
+      ```
+
+2. Deploy the DaemonSet by running:
```

Comment:
In my opinion, you don't need "by running:. Just put a period after "DaemonSet".